fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/data/CommandProcessingResultBuilder.java

```diff
@@ -106,6 +108,16 @@ public CommandProcessingResultBuilder withProductId(final Long productId) {
         return this;
     }
 
+    public CommandProcessingResultBuilder withGsimId(final Long gsimId) {
```

Comment:
This would have been painful during writing test cases and cause of many NPEs, I see gsimID and glimID added in commandProcessingResult result but in commandWrapper, we are mapping glimID and gsimID to entityID.
This has made it asymmetric and doesn't go well with Test infra. If possible take these new  instance variables out.